cns/NetworkContainerContract.go

```diff
@@ -29,6 +29,8 @@ const (
 	PathDebugIPAddresses                     = "/debug/ipaddresses"
 	PathDebugPodContext                      = "/debug/podcontext"
 	PathDebugRestData                        = "/debug/restdata"
+	NumberOfCPUCores                         = "/hostcpucores"
+	NMAgentSupportedAPIs                     = "/network/nmagentsupportedapis"
```

Comment:
these consts already exist already in [`api.go`](https://github.com/Azure/azure-container-networking/blob/master/cns/api.go#L34)

Reply:
Yeah, I was of two minds on this one. These seemed like `consts` for the client's consumption. I didn't like the duplication, so I'm more than happy to use the ones in `api.go` :)

Reply:
I think we should only have one instance of these consts that there is no chance that the server and client can drift apart. WDYT about just redefining these (in whichever direction prevents import cycles)?
eg `NumberOfCPUCores = api.NumberOfCPUCoresPath`

Reply:
@rbtr I changed it over to that, and I like it better since they at least can't diverge which is 💯 . Turns out they're both in the same package too, so yay no import cycles... but it kind of makes this second block's existence a little more dubious. I'm okay with letting it live for now if ultimately there's a future where we just generate this client entirely from swagger docs.